weather.noaa/default.py

```diff
@@ -0,0 +1,776 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from future import standard_library
+standard_library.install_aliases()
+
+import os, sys, time
+import xbmc, xbmcgui, xbmcaddon
+
+
+#from datetime import datetime
+from resources.lib.utils import FtoC, set_property, clear_property, log
+from resources.lib.utils import WEATHER_CODES, FORECAST, FEELS_LIKE, SPEED, WIND_DIR, SPEEDUNIT, zip_x
+from resources.lib.utils import get_url_JSON, decode_utf8  #, encode_utf8
+from resources.lib.utils import get_month, get_timestamp, get_weekday, get_time
+
+
+
+ADDON           = xbmcaddon.Addon()
+ADDONNAME       = ADDON.getAddonInfo('name')
+ADDONID         = ADDON.getAddonInfo('id')
+CWD             = decode_utf8(ADDON.getAddonInfo('path'))
```

Comment:
decode_utf8 shouldn't be needed here

Reply:
It breaks in Leia if the path isn't decoded to utf8.  (the decode_utf8() is a function to check the python version to determine whether-or-not to decode the string).  I'm not sure how to handle it without running that check.